pkg/remote/google/google_compute_network_enumerator.go

```diff
@@ -0,0 +1,50 @@
+package google
+
+import (
+	"strings"
+
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/remote/google/repository"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/google"
+)
+
+type GoogleComputeNetworkEnumerator struct {
+	repository repository.AssetRepository
+	factory    resource.ResourceFactory
+}
+
+func NewGoogleComputeNetworkEnumerator(repo repository.AssetRepository, factory resource.ResourceFactory) *GoogleComputeNetworkEnumerator {
+	return &GoogleComputeNetworkEnumerator{
+		repository: repo,
+		factory:    factory,
+	}
+}
+
+func (e *GoogleComputeNetworkEnumerator) SupportedType() resource.ResourceType {
+	return google.GoogleComputeNetworkResourceType
+}
+
+func (e *GoogleComputeNetworkEnumerator) Enumerate() ([]*resource.Resource, error) {
+	resources, err := e.repository.SearchAllNetworks()
+	if err != nil {
+		return nil, remoteerror.NewResourceListingError(err, string(e.SupportedType()))
+	}
+
+	results := make([]*resource.Resource, 0, len(resources))
+
+	for _, res := range resources {
+		results = append(
+			results,
+			e.factory.CreateAbstractResource(
+				string(e.SupportedType()),
+				strings.Replace(res.Name, "//compute.googleapis.com/", "", 1),
```

Comment:
`strings.TrimPrefix`

Reply:
Maybe we can make an unexported func to mutualize the logic with the `compute_firewall`

Reply:
Yes, this will be done in #1067